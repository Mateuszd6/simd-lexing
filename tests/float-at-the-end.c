float foobar = 123.3123213213