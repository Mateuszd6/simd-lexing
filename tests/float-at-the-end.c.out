./tests/float-at-the-end.c:1:1: TOK 0 (L = 5): "float"
./tests/float-at-the-end.c:1:7: TOK 0 (L = 6): "foobar"
./tests/float-at-the-end.c:1:14: TOK 1 (L = 1): "="
./tests/float-at-the-end.c:1:16: TOK 3 (L = 14): "123.3123213213"
Parsed: 1 lines, 2 ids, 0 strings, 0 chars, 0 ints, 0 hex,   1 floats,    0 //s, 0 /**/s,   0 #foo
